### PR TITLE
Add VITAE

### PR DESCRIPTION
**Pull request by jaydu1**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jaydu1** [comment]
@conda-forge/help-python, ready for review!

File: recipes/pyvitae/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/pyvitae/meta.yaml:1]: As it says here, please remove comments not specific to the package you are submitting here.

File: recipes/pyvitae/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # More info about selectors can be found in the conda-build docs: 
+  # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pyvitae/meta.yaml:43]: It doesn't look like this package compiles C-extensions. If so, then you can drop the `build` section under `requirements`.

File: recipes/pyvitae/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+  sha256: f41849ccb8804428830082e82505e03d020acc6a14d0a1a8edabedef74531f3d
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
```
Comment (synapticarbors) [inline recipes/pyvitae/meta.yaml:28]: I think this package can be `noarch: python`. See:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

If that's the case, remove the `skip: True` line and change the `python` requirements to `- python >=3.7`.

File: recipes/pyvitae/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}    
+  host:
```
Comment (synapticarbors) [inline recipes/pyvitae/meta.yaml:47]: Typically under `host` you only need `python` and `pip`. The rest are only needed to run your package and not required in the build stage.

File: recipes/pyvitae/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - VITAE
+  # For python packages, it is useful to run pip check. However, sometimes the
+  # metadata used by pip is out of date. Thus this section is optional if it is
+  # failing.
+  # requires:
```
Comment (synapticarbors) [inline recipes/pyvitae/meta.yaml:92]: I would include the `pip check` test.

**Review by synapticarbors — CHANGES_REQUESTED** [review]
In addition to my comments, you shouldn't delete or modify other files that are in `staged-recipes` related to other packages.

**Comment by synapticarbors** [comment]
It looks like conda-forge never had tensorflow==2.3. You can see all of the available versions here:
https://anaconda.org/conda-forge/tensorflow/files

or using `mamba repoquery search tensorflow`

**Comment by jaydu1** [comment]
Hi @synapticarbors , thanks for reviewing the code. I have tried to resolve most of the problems, but still cannot make it work  on the win-64 platform. 

The Azure log file says there is no `tensorflow >= 2.4` available. And I found `tensorflow>=2` is only available in the conda main channel:
`conda search  "tensorflow>=2.4" --subdir win-64 -c main`

Do you know any way to include the conda main cannel to the recipe? Thanks in advance.

> It looks like conda-forge never had tensorflow==2.3. You can see all of the available versions here: https://anaconda.org/conda-forge/tensorflow/files
> 
> or using `mamba repoquery search tensorflow`

**Comment by synapticarbors** [comment]
It looks like conda-forge doesn't build Tensorflow for windows. There are several complicated packages that we've punted on the Windows build for one reason or another (the same is true for Pytorch). We can ignore that here since this is noarch python. Users will still have the option to get tensorflow through another channel. 

If you could just fix the issue where you touched or deleted other files outside of the package you were working on, I'd be happy to merge this.

**Comment by jaydu1** [comment]
Thanks for the prompt reply! I think I have fixed the issues you mentioned above, except I didn't include the `pip check` test, because it does not correctly identify some conda dependencies.

**Comment by jaydu1** [comment]
Hi @synapticarbors , is there any other change I need to make? Otherwise could u help to merge it? Many thanks!

**Comment by synapticarbors** [comment]
There's just the issue of backing out changes to recipes not related to your specific package. After you do that, I'm happy to merge this PR.

**Pull request closed without merging**